From 9ac8ab83db40e35f4a65d20dc242e04aac017535 Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Wed, 2 Mar 2016 17:30:25 +0200
Subject: [PATCH 0920/1240] menuconfig: mvebu: menuconfig clean up - mvebu
 tools

moved tools support next to its enabling config.

Change-Id: I6c35988b6d062d7b46b5f3bf3b0867cc867ec34f
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27997
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm/cpu/mvebu-common/Kconfig | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/cpu/mvebu-common/Kconfig b/arch/arm/cpu/mvebu-common/Kconfig
index a2bc066..aaef785 100644
--- a/arch/arm/cpu/mvebu-common/Kconfig
+++ b/arch/arm/cpu/mvebu-common/Kconfig
@@ -107,6 +107,10 @@ config MVEBU_TOOLS_SUPPORT
 	  for different tools function for
 	  U-Boot/SPL code
 
+if MVEBU_TOOLS_SUPPORT
+source "arch/arm/cpu/mvebu-common/tools/Kconfig"
+endif
+
 config MVEBU_SYS_INFO
 	bool "System info transmitted to u-boot"
 	default y
@@ -125,7 +129,3 @@ config MVEBU_ROM_SILENCE_FLAG_SUPPORT
 	  silence parameter passed to SPL by
 	  BootROM for suppressing serial all
 	  port output during SPL run.
-
-if MVEBU_TOOLS_SUPPORT
-source "arch/arm/cpu/mvebu-common/tools/Kconfig"
-endif
-- 
1.9.1

